contributing_to_docs/doc_guidelines.adoc

```diff
@@ -217,8 +217,77 @@ Considering that we use distinct branches to keep content for product versions s
 
-*Do not use internal company server names in command or example output*. See suggested host name examples https://docs.openshift.com/container-platform/3.11/install/example_inventories.html#multi-masters-single-etcd-using-native-ha[here].
+== Node name formatting
 
+Do not use internal company server names in commands or example output. Provide generic OpenShift Container Platform node name examples that are not provider-specific, unless required. Where possible, use the example.com domain name when providing fully qualified domain names (FQDNs).
+
+The following table includes example OpenShift Container Platform 4 node names and their corresponding role types:
+
+[options="header"]
+|===
+
+|Node name |Role type
+
+|*node-1.example.com*
+.3+.^|You can use this format for nodes that do not need role-specific node names.
+
+|*node-2.example.com*
+
+|*node-3.example.com*
+
+|*control-plane-1.example.com*
+.3+.^|You can use this format if you need to describe the control plane role type within a node name.
+
+|*control-plane-2.example.com*
+
+|*control-plane-3.example.com*
+
+|*compute-1.example.com*
+.2+.^|You can use this format if you need to describe the compute node role type within a node name.
+
+|*compute-2.example.com*
+
+|*bootstrap.example.com*
+|You can use this format if you need to describe the bootstrap node role type within a node name.
+|===
+
+This example lists the status of cluster nodes that use the node name formatting guidelines:
+
+....
+[source,terminal]
+----
+$ oc get nodes
+----
++
+.Example output
+[source,terminal]
+----
+NAME                          STATUS   ROLES    AGE   VERSION
+compute-1.example.com         Ready    worker   33m   v1.19.0+9f84db3
+control-plane-1.example.com   Ready    master   41m   v1.19.0+9f84db3
+control-plane-2.example.com   Ready    master   45m   v1.19.0+9f84db3
+compute-2.example.com         Ready    worker   38m   v1.19.0+9f84db3
+compute-3.example.com         Ready    worker   33m   v1.19.0+9f84db3
+control-plane-3.example.com   Ready    master   41m   v1.19.0+9f84db3
+----
+....
+
+[NOTE]
+====
+Some provider-formatted host names include IPv4 addresses. An OpenShift Container Platform node name typically reflects the host name of a node. If node names in your output need to be provider-specific and require this format, use IPv4 addresses in the 192.0.2.0 to 192.0.2.255 reserved address range. For example, you could use `ip-192-0-2-1.example.com` as a node name that includes an IPv4 address.
+====
+
+== IPv4 address formatting
+
+Where possible, use the default OpenShift Container Platform private IPv4 addresses in example output. Private IPv4 addresses are those that are included in the following reserved address ranges:
+
+* 10.0.0.0 to 10.255.255.255
```

Comment:
For this, I'd include 10/8, 172.16/12, 192.168/16.

Reply:
And perhaps class A, B, C as well.

Reply:
@jboxman thank you for these suggestions. I have now updated the list.